### PR TITLE
Simone sandbox

### PR DESCRIPTION
**Pull request by 5im1z**
made changed according to comments in the prev PR (uncommented certain lines, added back mistakenly removed files, nbstripout'ed notebooks, etc.)

### CODE REVIEW

**Comment by javadnoorb** [comment]
@5im1z can you also resolve the merge conflicts? -thanks

**Review by javadnoorb — APPROVED** [review]
This looks good as far as I can tell

File: depmapomics/config_prod.py [hunk 1]
```diff
@@ -93,6 +92,10 @@
 
 ## RUN SPECIFIC
 
+VIRTUAL_FOLDER = "8d9c4c0691154a1f86b1b6e67c3fb683"
```
Comment (jkobject) [inline depmapomics/config_prod.py:95]: already above

**Review by jkobject — CHANGES_REQUESTED** [review]
Great, no real changes anymore it seems...

Maybe don't drop the csv and txt files from james_rna, this is the only location where I have that data and this is for a paper around 2 years ago I think this is so that people are able to get it and reproduce results.

**Pull request merged**